### DISCUSSION TITLE
Lista de empresas de tecnologia em floripa

### DISCUSSION THREAD

**Discussion by thiamsantos** [frontendbr/forum#1011] [Discussão]
Hey pessoal, eu estou começando uma lista com todas as empresas de tecnologia de floripa e região. Quem é de Floripa ou região bora lá ajudar. https://github.com/thiamsantos/floripa-tech-companies

**Comment by felipefialho** [comment]
@diogomoretti

**Comment by diogomoretti** [comment]
Fala @thiamsantos, depois vou colocar algumas empresas ali... Seria legal depois colocar o bairro e cidade.

Outra coisa: Você ainda tá morando em Palhoça? Em breve teremos algumas novidades por lá =]

**Comment by thiamsantos** [comment]
> Fala @thiamsantos, depois vou colocar algumas empresas ali... Seria legal depois colocar o bairro e cidade.

Opa, vlw @diogomoretti. Realmente seria interessante informações de bairro e cidade. Eu tava pensando também em adicionar algum tipo de classificação por segmento de trabalho.

> Outra coisa: Você ainda tá morando em Palhoça? Em breve teremos algumas novidades por lá =]

Fiquei curioso hehe.

**Comment by felipemfp** [comment]
Cara, não seria interessante agregar nesse [dinhani/empresas-tecnologia-informacao](https://github.com/dinhani/empresas-tecnologia-informacao), não?

**Comment by thiamsantos** [comment]
> Cara, não seria interessante agregar nesse dinhani/empresas-tecnologia-informacao, não?

Eu não conhecia esse projeto, interessante. Acho que um não exclui o outro, mas vou tentar contribuir.
